client/store.js

```diff
@@ -1,13 +1,23 @@
 import {createStore, applyMiddleware} from 'redux';
 import reducer from './reducers';
 import thunk from 'redux-thunk';
+import { composeWithDevTools } from '@redux-devtools/extension';
 
 const middlewares = [thunk];
 
 export default function configureStore(initialState = {}) {
+
+  let middleware;
+
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
new entry to `.env.example`
NODE_ENV === 'development'

Reply:
that's in different repo right? - asl-internal-ui

I guess I raise another PR for it?

Reply:
There will need to be a cascade of PRs - so after this there will need to be a commit to asl-pages bumping the asl-projects version, then to asl and asl-internal-ui bumping the pages and projects versions. You can update the .example.env at the same time.